contracts/hackatom/tests/integration.rs

```diff
@@ -16,22 +16,22 @@
 //!          //...
 //!      });
 //! 4. Anywhere you see query(&deps, ...) you must replace it with query(&mut deps, ...)
-//! 5. When matching on error codes, you can not use Error types, but rather corresponding ApiError variants.
+//! 5. When matching on error codes, you can not use Error types, but rather corresponding StdError variants.
```

Comment:
Actually, I think these two sides are identical now.
Impressive!

Please double-check and update

Reply:
Done in https://github.com/CosmWasm/cosmwasm/pull/305/commits/fc102963f711f3164b4efa6cc7230fc7f44f6f62